oss_src/unity/python/sframe/__init__.py

```diff
@@ -62,6 +62,8 @@
 from .version_info import version
 from .version_info import __VERSION__
 
+from _gl_pickle import GLPickler
```

Comment:
For PY3 support, you'll also need to always use relative imports:

```
from ._gl_pickle import GLPickler
from ._gl_pickle import GLUnpickler
```

Notice the periods.

Looks like you'll also need to make a similar change in `_gl_pickle.py`.